hooks/post-command

```diff
@@ -40,12 +40,20 @@ if [[ -z $FACTORY_COMMAND ]]; then
 fi
 echo "Using factory command: $FACTORY_COMMAND"
 
+step_key=$(buildkite-agent step get "key" || echo "")
+echo "Buildkite step key: '$step_key'"
+step_outcome=$(buildkite-agent step get "outcome" --step "$step_key" || echo "")
```

Comment:
Wasn't it simply missing the variable already available? 😊 
```suggestion
step_outcome=$(buildkite-agent step get "outcome" --step "${BUILDKITE_STEP_KEY}" || echo "")
```

Reply:
Sure, will fix in a new PR. Forgot to check the set of env vars in bk.